tests/src/JIT/SIMD/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required (VERSION 2.6)
+project (Vector3TestNative)
+include_directories(${INC_PLATFORM_DIR})
+set(SOURCES Vector3TestNative.cpp )
+
+# add the executable
+add_library (Vector3TestNative SHARED ${SOURCES})
+
+# add the install targets
+install (TARGETS Vector3TestNative DESTINATION bin)
```

Comment:
Some of the other CMakeLists.txt files have a set_target_properties at the end with a comment claiming it's necessary for System V systems.  I'm not sure that's the case, but I just wanted to make sure you've confirmed this tests will work on non-Windows.